### DISCUSSION TITLE
Subtitle support?

### DISCUSSION THREAD

**Discussion by boognish43** [chrisbenincasa/tunarr#529] [General]
Hi there, just discovered tunarr as I was setting up a new unraid system and went to install dizquetv as usual. This is looking great and everything so far is working well, great work! I'm wondering if I'm missing an option to turn on subtitles, or if perhaps that's not yet available. Thanks so much

**Comment by chrisbenincasa** [comment]
Thanks! We removed interaction with the Plex transcoder, which is how dizquetv originally supported subtitles. The rationale for removing that can be found on the PR: https://github.com/chrisbenincasa/tunarr/pull/452

Unfortunately this had the side effect of removing subtitle support. We have https://github.com/chrisbenincasa/tunarr/issues/462 to track a more fully fledged implementation of that, but it's a non trivial undertaking. I think a quick win there would be to start with external subtitles (not embedded) since they are simpler to deal with in the new streaming setup. 

Sorry for the inconvenience there, but we will definitely get them back!

**Reply by boognish43** [reply]
thanks for the thorough reply! Totally understand the complexity of the work and will just keep my eyes out for this in the future. Thanks again for the reply and especially for all the work on this project <3

**Comment by theoisadoor** [comment]
Any update on this? I don't seem to be able to display subtitles that are embedded for a foreign show. Even though subtitles are enabled, none of them are showing.

**Reply by chrisbenincasa** [reply]
Subtitles are supported. If you're having an issue we'll need to look at your specific setup, with details like tunarr debug logs and media info. 

Embedded text based subtitles need to be extracted by tunarr (a global setting)
